starboard/shared/starboard/media/codec_util_test.cc

```diff
@@ -278,6 +278,21 @@ TEST(CodecUtilTest, ParsesIamfCodec) {
             kSbMediaAudioCodecIamf);
   EXPECT_EQ(GetAudioCodecFromString("iamf.000.000.ipcm", ""),
             kSbMediaAudioCodecIamf);
+  EXPECT_EQ(GetAudioCodecFromString("iamf.001.000.ipcm", ""),
+            kSbMediaAudioCodecIamf);
+  EXPECT_EQ(GetAudioCodecFromString("iamf.255.255.ipcm", ""),
+            kSbMediaAudioCodecIamf);
+
+  EXPECT_NE(GetAudioCodecFromString("iamf.000.256.Opus", ""),
```

Comment:
nit EXPECT_EQ(.., kSbMediaAudioCodecNone)?  And maybe add a comment in front of the group mentioning that they are invalid codecs.